README.md

```diff
@@ -1,63 +1,172 @@
-   
-
-### Additional Data
-These datasets are just seven of many that our project maintains. The rest are posted on our Google Sheets link here: https://docs.google.com/spreadsheets/d/1X6uJkXXS-O6eePLxw2e4JeRtM41uPZ2eRcOA_HkPVTk/edit#gid=1641553906
-
-Visit the above link to check out the following other datasets:
-	
-* COVID-19 Correctional Pop. Reduction Requests: Requests for authoritiative actions necessary to reduce correctional population densities
-* COVID-19 Correctional Pop. Reduction Responses: System responses and authoritative actions to reduce correctional population densities	
-* COVID-19 Related Legal Filings & Court Orders: Tracking specific legal requests and court orders			
-* COVID-19 Visitation Policy by State (Prisons): Tracking visitation suspension policies and offerings of compensatory remote access			
-* COVID-19 Jail/Prison Condition Policies: Descriptive summaries of ongoing policies affecting carceral conditions			
-* COVID-19 Fundraisers & Mutual Aid Efforts: Links and descriptions of COVID-19 fundraising efforts			
-* Other COVID-19 Organizing Efforts: Other organized efforts and grassroots campaigns			
-* Additional Resources: Collection of additional useful resources, recommendations, and databases			
+
+[![logo](README_files/logo.svg)](https://uclacovidbehindbars.org/)
+
+# UCLA Law COVID-19 Behind Bars Data
+
+## Background
+
+The [UCLA Law COVID-19 Behind Bars Data
+Project](https://uclacovidbehindbars.org/), launched in March 2020,
+tracks the spread and impact of COVID-19 in American carceral facilities
+and advocates for greater transparency and accountability around the
+pandemic response of the carceral system. Since March, we have been
+collecting and reporting facility-level data on COVID-19 in prisons,
+jails, and other correctional centers.
+
+We also collect information about pandemic-related prison and jail
+releases, legal filings and court orders bearing on the safety of
+incarcerated people, and grassroots organizing campaigns and fundraisers
+[here](https://docs.google.com/spreadsheets/u/2/d/1X6uJkXXS-O6eePLxw2e4JeRtM41uPZ2eRcOA_HkPVTk/edit#gid=1641553906).
+
+## Our Process
+
+Our core dataset includes information on COVID-19 cases, deaths, and
+tests across more than 1,500 state, federal, and county facilities. We
+maintain this dataset by scraping and standardizing data from more than
+80 sources. We scrape this data 3-4 times each week, although
+correctional agencies vary in how often they update their data. Our
+[scraper production
+code](https://github.com/uclalawcovid19behindbars/covid19_behind_bars_scrapers)
+and [more detailed
+documentation](https://github.com/uclalawcovid19behindbars/covid19_behind_bars_scrapers)
```

Comment:
Should this be a link to the README? Or the folder with the scrapers themselves? Think if we're going to hyperlink two separate things in this sentence, they should lead to different paths.

Reply:
Ahh good catch! The latter should be for the docs, I'll update now!